### ISSUE TITLE
Quick create similar

### ISSUE THREAD

**Issue by cjrobe** [QTodoTxt/QTodoTxt#88]
I think there should be an option "create new task based on selected"

So if you're viewing your shopping list, it would automatically add +shopping when you press insert to create a new task. If you're viewing all the tasks, it wouldn't add anything. Would be great for adding lots of similar tasks at once.

**Comment by celmi** [comment]
I'd love to have that feature. I often work project-focused, and going through the to-do list of a project I'dlike to add multiple to-do's quickly. It would be very cool not to reenter the project-markup every time.

Instead of / additionally to / an option "create new task based on selected" I envision a shortcut like CTRL-SHIFT-N which would automatically insert the context-markup or project-markup for the category currently chosen in the sidebar. 

I'm generally using the autofilters "Projects" and "Context" already provided by the software in the sidebar, and for me an autocomplete-function providing the markup for the currently filtered Project/Context would be good enough. So CTRL-SHIFT-N should be auto-inserting markup based on the sidebar-filter, not based on the currently selected Item.

(Maybe "create new task based on selected" then could auto-insert all Context/Project-Markup of the selected item, as an alternative, more detailed feature).

**Comment by anacondaqq** [comment]
maybe better add repeatable tasks?

**Comment by topas-rec** [comment]
Your feature request is already filed under: https://github.com/QTodoTxt/QTodoTxt/issues/255

**Comment by topas-rec** [comment]
You could comment there to support this feature.

**Comment by celmi** [comment]
No, Sorry, this is not the same as repeatable tasks! This feature request here is all about effortlessly creating SIMILAR tasks (e.g. same project- or same context-markup or both) on the fly. It's not about entering the same task over and over again.

**Comment by topas-rec** [comment]
Sorry for being not exact enough:
I meant that the alternative that anacondaqq mentioned ("maybe better add repeatable tasks?") is already a feature request filed under #255.

**Comment by topas-rec** [comment]
Your feature, celmi, would be nice to see in QTodoTxt. +1
(Simpletasks on android allows you to do this already)
